feature-libs/product/future-stock/components/future-stock/future-stock.component.html

```diff
@@ -0,0 +1,7 @@
+<ng-container *ngIf="futureStocks">
+  <cx-future-stock-accordion
+    [header]="'futureStockDropdown.header' | cxTranslate"
```

Comment:
We shouldn't need to pass this translation key if its always the same for a future stock-specific component.

Reply:
good point, removed header as input, it's now translated directly in accordion